src/lua/init.c

```diff
@@ -1164,6 +1164,17 @@ run_script_f(va_list ap)
 			lua_settop(L, 0);
 			break;
 		case 'e':
+			/*
+			 * Do not run chunk given via -e option if
+			 * integrity check is enabled.
+			 * XXX: Fortunately, -e and its argument
+			 * are stripped from Lua <arg> table, so
+			 * this is "last homely house" where one
+			 * can obtain the given Lua chunk.
```

Comment:
I'm a little bit concerned about the silent skip of `-e` options -- it may not be obvious to the user. Please add the warning printing or at least a `TODO` mark about it.

Reply:
Added.

```diff
diff --git a/src/lua/init.c b/src/lua/init.c
index cbe3d9f816..4f9dc608b1 100644
--- a/src/lua/init.c
+++ b/src/lua/init.c
@@ -1171,6 +1171,8 @@ run_script_f(va_list ap)
                         * are stripped from Lua <arg> table, so
                         * this is "last homely house" where one
                         * can obtain the given Lua chunk. 
+                        * TODO: inform user that -e Lua chunk was
+                        * not executed during the startup.
                         */
                        if (integrity)
                                break;
                                ```

Reply:
Typo: s/user/the user/

Reply:
Fixed (again).